src/odemis/model/_metadata.py

```diff
@@ -178,6 +178,9 @@
 # dict (int (resolution X) -> dict (float (dwell time) -> tuple of 4 floats (correction factors)))
 MD_SHIFT_LOOKUP = "Pixel shift compensation table"
 
+# The following metadata is used for the calibration of the tfsbc.BeamShiftController
+MD_CALIB_BEAMSHIFT = "Beamshift calibration"  # 4x2 tuple, 4x (float, float), xlower, ylower, xupper, yupper
```

Comment:
We now have a generic `MD_CALIB`, which you can use in your class. Please document well xlower, ylower, xupper, yupper in the docstring of the class.
You should be able to drop entirely the changes on this file.